recipes/igblast/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+c_compiler:            # [osx]
+  - clang              # [osx]
+c_compiler_version:    # [osx]
+  - 9                  # [osx]
+cxx_compiler:          # [osx]
+  - clangxx            # [osx]
+cxx_compiler_version:  # [osx]
+  - 9                  # [osx]
```

Comment:
I add this because a quick web search for
```
... install_name_tool: object: ... malformed object (unknown load command 9)
```
indicated this means the `install_name_tool` was too old to handle the binaries.